float32 highest_temp
float32 lowest_temp
float32[] tempf
int32[] tempui
int32 highest_temp_id
int32 lowest_temp_id